app/src/main/java/org/mozilla/focus/home/HomeFragment.java

```diff
@@ -105,31 +107,25 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         updateTabCounter();
 
         this.fakeInput = (TextView) view.findViewById(R.id.home_fragment_fake_input);
-        this.fakeInput.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
+        this.fakeInput.setOnClickListener(v -> {
                 final Activity parent = getActivity();
                 if (parent instanceof FragmentListener) {
                     ((FragmentListener) parent).onNotified(HomeFragment.this,
                             FragmentListener.TYPE.SHOW_URL_INPUT,
                             null);
                 }
                 TelemetryWrapper.showSearchBarHome();
-            }
         });
 
         SwipeMotionLayout home_container = (SwipeMotionLayout) view.findViewById(R.id.home_container);
-        home_container.setOnSwipeListener(new OnSwipeListener.OnSwipeListenerAdapter() {
-            @Override
-            public void onSwipeUp() {
-                btnMenu.performClick();
-            }
+        home_container.setOnSwipeListener(new GestureListenerAdapter());
 
-            @Override
-            public void onSwipeDown() {
-                fakeInput.performClick();
-            }
-        });
+        if (ThemeManager.shouldShowOnboarding(view.getContext())) {
+            LayoutInflater.from(view.getContext()).inflate(R.layout.fragment_homescreen_themetoy, home_container);
```

Comment:
Maybe themeOnboardingLayer = LayoutInflater.from(view.getContext()).inflate(R.layout.fragment_homescreen_themetoy, home_container); ?

Reply:
The return value of the ```inflate``` is the second param, which is the ```home_container``` if it is not ```null```. What I need is the ```themeOnboardingLayer```. When I pass ```home_container```, the inflated view will be add to the root also. so I can find the view with the id ```themeOnboardingLayer``` later.